
Hi,

Just this morning I figured out that gpg-agent does not anymore close
all files.  This is in particular annoying if you use ssh along with the
agent.  The patch is simple:


	* estream.c (es_func_file_create): Clear NO_CLOSE flag.

--- common/estream.c    (revision 4541)
+++ common/estream.c    (working copy)
@@ -861,6 +861,7 @@
 #endif
 
   file_cookie->fd = fd;
+  file_cookie->no_close = 0;
   *cookie = file_cookie;
   *filedes = fd;



